tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -147,11 +147,11 @@ export const UserDetails = (props) => {
         </Stack.Item>
         <Stack.Item>
           <LabeledList>
-            <LabeledList.Item label="User">
+            <LabeledList.Item label="Сотрудник">
               {checking_id ? user.name : 'UNKNOWN'}
             </LabeledList.Item>
-            <LabeledList.Item label="Occupation">
-              {checking_id ? user.job || 'Unemployed' : 'UNKNOWN'}
+            <LabeledList.Item label="Должность">
+              {checking_id ? (user.job ? capitalize(user.job) : user.job) || 'Unemployed' : 'UNKNOWN'}
```

Comment:
А зачем капиталайзим?
```suggestion
              {checking_id ? capitalize(user.job) || 'Unemployed' : 'НЕИЗВЕСТНО'}
```

Reply:
Т.к. в переводе ru_names.toml
будут названия должностей с маленькой буквы, не очень красиво выглядит в вендоре.
ФИО то с большой будут

Reply:
Должности не пихай в ru_names.toml, у нас отдельный код для перевода должностей есть. Просто где-то недокинули русское

Reply:
Поправил